SkyEditor.RomEditor.Rtdx/Infrastructure/CodeGenerationHelper.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Linq;
+using SkyEditor.RomEditor.Domain.Rtdx.Structures.Executable;
```

Comment:
Not sure how this domain and infrastructure pattern works, but I guess you maybe shouldn't import domain stuff in infrastructure? Let me know if there's a better way to structure this.

Reply:
At a high level, Domain contains all the stuff dealing with the primary subject matter (ROM editing), while Infrastructure contains all the helper code not specifically related to the domain.

Might as well just go ahead and move it to Domain if in doubt. Frankly, I should move Modpack stuff to Domain too.